### ISSUE TITLE
Would the Classification Results be comparable to DINOv3 paper?

### ISSUE THREAD

**Issue by J-am-ack** [sovit-123/dinov3_stack#15]
Hi~ Thanks a lot for your great work!
I'm using this repository to reproduce the results in Table 14 in DINOv3 paper, but the accuracy metrics seems to reach about 83%, and stagnate at that point.
But the paper claims that it could perform an accuracy about 88% on IN1k. I'm wondering whether there's any difference. Have you already apllied the several methods in the paper? (Like Gram Anchoring, RoPE Jittering, or any detailed training settings)

Looking forward to your reply and more discussion!~

Every response may be helpful!
